app/helpers/pouchDB.js

```diff
@@ -79,6 +89,67 @@ runMigration(
   }
 );
 
+// Get PouchDB Invoices (Used for Exporting)
+const pouchDBInvoices = () => {
```

Comment:
We don't we use [`getAllDocs`](https://github.com/hql287/Manta/blob/dev/app/helpers/pouchDB.js#L101-L115) instead? 🧐

Reply:
Looks like you need this to return different kind of doc than `getAllDocs`?

Reply:
Once again I didn't do enough testing :disappointed: I tried to use it in the beginning and saw the end result being different, so I just assumed it would be wrong to use. But now that I can compare `getAllDocs('invoices')` and `pouchDBInvoices` I see the only difference is that I have to wrap `getAllDocs` with an array and that it is missing the `_revisions` object wich is not important.

**I will use `getAllDocs` instead** :slightly_smiling_face: **And remove the extra `pouchDBInvoices` and `pouchDBContacts` helpers which will reduce dependencies!**

Thank you!